views_test.go

```diff
@@ -184,15 +190,17 @@ func TestCreateTest(t *testing.T) {
 
 	buildID := "5a75f537726934e4b62833ab6d5dca41"
 	type payload struct {
-		TestFilename string `json:"test_filename"`
-		Command      string `json:"command"`
-		Phase        string `json:"phase"`
-		TaskID       string `json:"task_id"`
+		TestFilename  string `json:"test_filename"`
+		Command       string `json:"command"`
+		Phase         string `json:"phase"`
+		TaskID        string `json:"task_id"`
+		TaskExecution int    `json:"execution"`
```

Comment:
Could you add an execution to the `NewTest` sub-test and check that it gets set there?

Reply:
Check again, please!